src/sections/Meshery/Features-Col/data/index.js

```diff
@@ -1,148 +1,212 @@
-const lifecycleFeatureData = {
-  name: "Lifecycle",
-  description: "Manage all your cloud native infrastructure",
-  link: "#",
-  btnText: "Discover your clusters",
-  services: [
-    {
-      content:
-        "Discover, configure, visualize, validate and troubleshoot your existing Kubernetes clusters",
-      className: "--on",
-    },
-    {
-      content:
-        "Deploy and operate any service mesh",
-      className: "--on",
-    },
-    {
-      content:
-        "Use known cloud native ",
-      className: "--on",
-    },
-    {
-      content:
-        "Validate each service mesh's compliance with Service Mesh Interface (SMI)",
-      className: "--on",
-    },
-  ],
-  count: {
-    value: 10,
-    description: "service meshes supported",
-  },
-};
+import { useStaticQuery, graphql } from "gatsby";
 
-const configurationFeatureData = {
-  name: "Configuration",
-  description: "Operate with best practices",
-  link: "#",
-  btnText: "Get Started",
-  services: [
-    {
-      content: "Built-in industry patterns and known best practices",
-      className: "--on",
-    },
-    {
-      content: "Apply custom configuration to your Kubernetes clusters",
-      className: "--on",
-    },
-    {
-      content: "Manage WebAssembly filters for Envoy",
-      className: "--on",
-    },
-    {
-      content: "Version your infrastructure configuration designs",
-      className: "--on",
-    },
-    {
-      content: "Visualize and dry-run your deployment with MeshMap",
-      className: "--on",
-    },
-    {
-      content: "Apply well-known service mesh patterns",
-      className: "--on",
-    },
-  ],
-  count: {
-    value: 1500,
-    description: "components",
-  },
-};
+const LifecycleFeature = () => {
+  const integrations = useStaticQuery(graphql`
+    query {
+      allMdx(
+        filter: {
+          fields: { collection: { eq: "integrations" } }
+          frontmatter: { published: { eq: true } }
+        }
+      ) {
+        nodes {
+          frontmatter {
+            title
+            status
+            category
+            integrationIcon {
+              childImageSharp {
+                gatsbyImageData(width: 500, layout: CONSTRAINED)
+              }
+              extension
+              publicURL
+            }
+            darkModeIntegrationIcon {
+              childImageSharp {
+                gatsbyImageData(width: 500, layout: CONSTRAINED)
+              }
+              extension
+              publicURL
+            }
+          }
+          fields {
+            slug
+          }
+        }
+      }
+    }
+  `);
```

Comment:
This query can be optimised to get the count of folders under the `integrations` folder. We don't need all these nodes that are being queried here, please update the query. Take reference from the docs/other code snippets.

Reply:
Yeah, you are right. I am on it